dockers/docker-lldp-sv2/Dockerfile.template

```diff
@@ -0,0 +1,23 @@
+FROM docker-base
+
+COPY deps /deps
+COPY python-wheels /python-wheels
+
+RUN apt-get update && apt-get install -y python-pip supervisor libbsd0 libevent-2.0-5 libjansson4 libwrap0 libxml2 libpci3 libperl5.20
+
+## Pre-install the fundamental packages
+## Install Python SSWSDK
+## Install LLDP Sync Daemon
+
+SED_DPKG
+
+RUN pip install /python-wheels/sswsdk-2.0.1-py2-none-any.whl && \
```

Comment:
Filenames will be autogenerated the same way as debian packages
I will do corresponding changes as soon as possible, but for now it is hardcoded